src/less/mixins.less

```diff
@@ -0,0 +1,19 @@
+// @see https://github.com/wikimedia/mediawiki/blob/8e7843197e305020f35ba883fcef39a83ad1ebce/resources/src/mediawiki.less/mediawiki.mixins.animation.less
+.animation( ... ) {
+	-webkit-animation: @arguments; // Chrome 4-42, Safari 4-8, Opera 15-29, Android 2.1-4.4.4
+	-moz-animation: @arguments; // Firefox 5-15
+	animation: @arguments; // Chrome 43+, Firefox 16+, IE 10+, Edge 12+, Safari 9+, Opera 12.10 & 30+, iOS 9+, Android 47+
+}
+
+.transform-rotate( @deg ) {
+	-webkit-transform: rotate( @deg );
+	-moz-transform: rotate( @deg );
+	transform: rotate( @deg );
+}
+
+// @see https://github.com/wikimedia/mediawiki/blob/37e1fffd827b483f805c6c5296155768a74a2bc6/resources/src/mediawiki.rcfilters/styles/mw.rcfilters.mixins.less
+.animation-delay( ... ) {
+	-webkit-animation-delay: @arguments; // Chrome 4-42, Safari 4-8, Opera 15-29, Android 2.1-4.4.4
+	-moz-animation-delay: @arguments; // Firefox 5-15
+	animation-delay: @arguments; // Chrome 43+, Firefox 16+, IE 10+, Edge 12+, Safari 9+, Opera 12.10 & 30+, iOS 9+, Android 47+
+}
```

Comment:
We shouldn't need these browser prefixes, hence I don't think we need these mixins, right?

Reply:
Eh. I have to say, those cost us nothing, and they're used all over MediaWiki. I'm a bit weary of taking them all out...? Are we 100% sure they work without the prefixes?